.coveragerc

```diff
@@ -249,6 +249,9 @@ omit =
     homeassistant/components/eight_sleep/*
     homeassistant/components/eliqonline/sensor.py
     homeassistant/components/elkm1/*
+	homeassistant/components/elmax/__init__.py
+    homeassistant/components/elmax/const.py
+    homeassistant/components/elmax/switch.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.
(Also is there a formatting issue?)

Reply:
You were right: line 252 was idented with tab, while the others with 4 spaces. Fixed now!
About the 100% coverage: I will try to catch up with that, but I'd like the review to proceed and complete even without it.